lib/index.js

```diff
@@ -189,17 +189,7 @@ export async function verifyCapabilityInvocation({
     maxDelegationTtl,
     suite
   });
-  // invocation target must match absolute url
-  let invocationTarget;
-  // do not simply check for a colon (`:`) to find a relative URL
-  // because some applications allow and pass non-conformant
-  // relative URLs with unencoded colons (and we accept those
-  // because it is common to do so despite non-conformance)
-  if(url.startsWith('https://')) {
-    invocationTarget = url;
-  } else {
-    invocationTarget = `https://${headers.host}${url}`;
-  }
+  const invocationTarget = url
```

Comment:
This will be a breaking change, instead, can we change the conditional to look for a `:` in `url` as a signal of an absolute vs. a relative URL -- and, if it is relative, only then apply the `https://...` prefix?

```suggestion
  // invocation target must match absolute url
  let invocationTarget;
  // do not simply check for a colon (`:`) to find a relative URL
  // because some applications allow and pass non-conformant
  // relative URLs with unencoded colons (and we accept those
  // because it is common to do so despite non-conformance)
  if(url.includes(':')) {
    invocationTarget = url;
  } else {
    invocationTarget = `https://${headers.host}${url}`;
  }
```

Reply:
Yes that's fine with me. Thanks for suggesting something. I figured we could do better than what I proposed, but what I proposed didn't break any tests.

I will accept this change, then add a test to make sure the relative path behavior does not accidentally regress later.

Reply:
In the future, it seems appropriate to maybe add a parameter for the scheme (or function to resolve relative href), which defaults to this behavior, so folks can use it with relative hrefs but not necessarily https: scheme. Personally I don't need that right now. May propose it later as needed.

Reply:
added tests in https://github.com/digitalbazaar/http-signature-zcap-verify/pull/39/commits/f8c989afc705caedcee9b8db429d99ebfbc0c986

Reply:
non urgent idea: it seems like there could be a bug if someone passes options.url value that does not contain ':' and does not start with '/'. fix later imho.